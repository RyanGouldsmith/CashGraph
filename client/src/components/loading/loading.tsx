import React from 'react';

export const Loading = React.memo<{}>(() => {
  return <h1>Loading</h1>;
});
