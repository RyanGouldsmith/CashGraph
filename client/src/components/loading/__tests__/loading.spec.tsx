import 'babel-polyfill';
import React from 'react';
import { render, wait, cleanup } from 'react-testing-library';

import { Loading } from '../loading';

afterEach(cleanup);

test('should match the snapshot for Loading', async () => {
  const { container } = render(<Loading />);

  await wait(() => {
    expect(container).toMatchSnapshot();
  });
});
