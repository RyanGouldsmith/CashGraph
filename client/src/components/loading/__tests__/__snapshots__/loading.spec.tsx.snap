// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot for Loading 1`] = `
<div>
  <h1>
    Loading
  </h1>
</div>
`;
